WordPress/Classes/ViewRelated/New Group/SiteCreationBuilder.swift

```diff
@@ -0,0 +1,12 @@
+// Tracks data state shared between Site Creation Wizard Steps. I am not too fond of the name, but it kind of works for now.
```

Comment:
>  I am not too fond of the name, but it kind of works for now.

`SiteCreator` maybe? The `build` method still underscores that it's a kind-of builder.